### PR TITLE
Second branch

### PR DESCRIPTION
**Pull request by Busiwa24**
In the following pull request I have done the following:
-added the projects section
-frameworks section
-linter error checks 
-My about section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -35,5 +38,137 @@ <h1 class="heading-1">
...
+            <img src="img/Rectangle26.png" alt="Rectangle" class="projects-img">
+            <h3 class="projects-heading-2">Project Name goes here</h3>
+            <ul class="projects-card">
+            <li class="project-list"><button class="list-btn" type="button">Ruby On Rauls</button></li>
+            <li class="project-list"><button class="list-btn" type="button">Ruby On Rauls</button></li>
+            <li class="project-list"><button class="list-btn" type="button">Javascript</button></li>
+        </ul>
+        <button class="project-button" type="button" >
+        <span>See This Project</span>
+        <span><ion-icon name="arrow-forward-outline"></ion-icon></span>
```
Comment (Emmyn5600) [inline index.html:124]: - [ ] The button (e.g., "See This project") is a link a or button tag nice you did it, Please make the button styling: Active, Hover and Pressed.

File: index.html [hunk 2]
```diff
@@ -35,5 +38,137 @@ <h1 class="heading-1">
...
+            <h3 class="projects-heading-2">Project Name goes here</h3>
+            <ul class="projects-card">
+            <li class="project-list"><button class="list-btn" type="button">Ruby On Rauls</button></li>
+            <li class="project-list"><button class="list-btn" type="button">Ruby On Rauls</button></li>
+            <li class="project-list"><button class="list-btn" type="button">Javascript</button></li>
+        </ul>
+        <button class="project-button" type="button">
+        <span>See This Project</span>
+        <span><ion-icon name="arrow-forward-outline"></ion-icon></span> 
+        </button>
```
Comment (Emmyn5600) [inline index.html:112]: - [ ] The button (e.g., "See This project") is a link a or button tag nice you did it, Please make the button styling: Active, Hover and Pressed.

File: index.html [hunk 3]
```diff
@@ -35,5 +38,137 @@ <h1 class="heading-1">
...
+            <img src="img/Rectangle24.svg" alt="Rectangle" class="projects-img">
+            <h3 class="projects-heading-2">Project Name goes here</h3>
+            <ul class="projects-card">
+            <li class="project-list"><button class="list-btn" type="button">Ruby On Rauls</button></li>
+            <li class="project-list"><button class="list-btn" type="button">Ruby On Rauls</button></li>
+            <li class="project-list"><button class="list-btn" type="button">Javascript</button></li>
+        </ul>
+        <button class="project-button" type="button">
+        <span>See This Project</span>
+        <span><ion-icon name="arrow-forward-outline"></ion-icon></span>
```
Comment (Emmyn5600) [inline index.html:98]: - [ ] The button (e.g., "See This project") is a link a or button tag nice you did it, Please make the button styling: Active, Hover and Pressed.

File: index.html [hunk 4]
```diff
@@ -35,5 +38,137 @@ <h1 class="heading-1">
...
+            <img src="img/Rectangle23.svg" alt="Rectangle" class="projects-img">
+            <h3 class="projects-heading-2">Project Name goes here</h3>
+            <ul class="projects-card">
+            <li class="project-list"><button class="list-btn" type="button">Ruby On Rauls</button></li>
+            <li class="project-list"><button class="list-btn" type="button">Ruby On Rauls</button></li>
+            <li class="project-list"><button class="list-btn" type="button">Javascript</button></li>
+        </ul>
+        <button class="project-button" type="button">
+        <span>See This Project</span>
+        <span><ion-icon name="arrow-forward-outline"></ion-icon></span>
```
Comment (Emmyn5600) [inline index.html:85]: - [ ] The button (e.g., "See This project") is a link a or button tag nice you did it, Please make the button styling: Active, Hover and Pressed.

File: index.html [hunk 5]
```diff
@@ -35,5 +38,137 @@ <h1 class="heading-1">
...
+        </button>
+        </div>
+    </section>
+    <section class="about">
+        <h1 class="heading-1">
+            About
+        </h1>
+        <p class="about-par">
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ea eiusmod
+        </p>
```
Comment (Emmyn5600) [inline index.html:134]: - [ ] Please some padding between h1 and paragraph as it is on the Figma design

File: index.html [hunk 6]
```diff
@@ -35,5 +38,137 @@ <h1 class="heading-1">
...
+    <section class="section-3">
+        
+        <div class="background-color">
+            <img src="img/Rectangle21.svg" alt="Rectangle" class="projects-img">
+            <h3 class="projects-heading-2">Project Name goes here</h3>
+            <ul class="projects-card">
+            <li class="project-list"><button class="list-btn" type="button">Ruby On Rauls</button></li>
+            <li class="project-list"><button class="list-btn" type="button">Ruby On Rauls</button></li>
+            <li class="project-list"><button class="list-btn" type="button">Javascript</button></li>
+        </ul>
```
Comment (Emmyn5600) [inline index.html:56]: - [ ] The main container of the projects has a display:grid, please display the container of the project as display:grid.

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
#### Hi @Busiwa24  :raised_hands:

Good job so far!:clap::clap:
We, however, have some issues to look into before we may move forward but you are almost there!:muscle:

##  Required Changes :recycle::warning: 
* Update your Live demo link that has both work and about sections

**_Check the comments under the review._**

## Optional suggestions

* Please try to make more commits in your project instead of pushing all changes at once. 

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: index.html (hunk 6 above)
Comment (Busiwa24) [reply index.html:56]: Thank you. Appreciate the feedback, issues have been handled

**Review by carlos-ssh — APPROVED** [review]
Hi @Busiwa24, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**